### PR TITLE
Bump BSI, specifically consistent-eval

### PR DESCRIPTION
**Pull request by AlexBedley**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4924,20 +4819,6 @@
         }
       }
     },
-    "d2l-facet-filter-sort": {
```
Comment (AlexBedley) [inline package-lock.json:4927]: hmm this seems wrong

File: package-lock.json [hunk 2]
```diff
@@ -5077,7 +4958,6 @@
     "d2l-localize-behavior": {
       "version": "github:BrightspaceUI/localize-behavior#ff9514e84470665348ccbb8584c32b5f51abf13b",
       "from": "github:BrightspaceUI/localize-behavior#semver:^2",
-      "dev": true,
```
Comment (AlexBedley) [inline package-lock.json:5080]: it also seems my version of npm hates the `dev` member

File: package-lock.json [hunk 3]
```diff
@@ -4506,7 +4506,7 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#645d0f95b99e9a32c93471f0c01013fc8617f330",
+      "version": "github:BrightspaceHypermediaComponents/activities#08a1e07ba6a3070f2bbe5177e4b6e61b719450e6",
```
Comment (AlexBedley) [inline package-lock.json:4509]: fyi @ChrisLabattD2L

**Comment by AlexBedley** [comment]
ok it looks much better now. my node version was too old I guess

**Review by JamieTran — APPROVED** [review]

**Pull request merged**